CorsixTH/Lua/world.lua

```diff
@@ -1242,7 +1242,7 @@ end
 -- during the coming month.
 function World:updateSpawnDates()
   -- Set dates when people arrive
-  local no_of_spawns = math.n_random(self.spawn_rate, 2)
+  local no_of_spawns = math.n_random(self.spawn_rate * self:getReputationImpact(self:getLocalPlayerHospital()), 2)
```

Comment:
Hm, the previous design thought has been that the population variable of the hospital class should be used to vary how many patients a hospital gets. Since we have no AI it is at first 25 % and then always 50 %. A map maker knows that (with some variance of course) the spawn rate for his or her "World" is a certain number, and then it's up to the player to get as many of those people as possible. Changing the total number of spawns based on the local player's reputation like this gets us even further painted into a corner?

It's just a matter of semantics, but I would prefer to move the local player hospital dependency down to the next row just below here, to have them in the same place, and to let the map maker dictate more closely the spawn rate.